metricbeat/module/apache/_meta/Dockerfile

```diff
@@ -1,6 +1,5 @@
 ARG APACHE_VERSION=${APACHE_VERSION}
 FROM httpd:$APACHE_VERSION
-RUN sed -i "/jessie-updates/d" /etc/apt/sources.list
-RUN apt-get update && apt-get install -y curl
+RUN apt update && yes | apt install curl
```

Comment:
This is different on purpose. `yes |` also allows to install packages without authentication in case of expired keys. The keys in Jessie are expired at the moment and we need this Debian version for testing old log format for httpd <= 2.4.12. So either we have to update the keys as one of the steps or just accept the prompt to install without authentication.